src/client/scripts/room.js

```diff
@@ -18,6 +18,14 @@ Template.room.helpers({
         ).categories;
     },
 
+    getNumCategories: function(){
```

Comment:
again getNumCategories needs to just return the number of categories.  If you need a method to do this, find a better name.